setup.cfg

```diff
@@ -235,10 +228,10 @@ source-dir = docs
 #
 # The gfortran_lib_dirs should be set if needed.
 #
-#xspec_version = 12.9.0
+#xspec_version = 12.10.1
 #xspec_lib_dirs = None
 #xspec_include_dirs = None
-#xspec_libraries = XSFunctions XSModel XSUtil XS
+#xspec_libraries = XSFunctions XSUtil XS hdsp_6.26  # change 6.26 to the correct version
```

Comment:
@Marie-Terrell had asked about this change. I have a two-pronged approach to building against XSPEC

a) try to follow the " Appendix F: Using the XSPEC Models Library in Other Programs " guide which is currently at https://heasarc.gsfc.nasa.gov/xanadu/xspec/manual/node323.html (they do have a stable URL for it but you can no-longer find it and I'm too lazy to go digging in our docs to try and find it)

which at this time says

```Makefile
myprog : myprog.o
	g77 -g myprog.o -o myprog \
		-L/path/to/headas/installed/location/lib \
		-lXSFunctions -lXSUtil -lXS -lhdsp_2.9 -lCCfits_2.5 -lcfitsio
myprog.o: myprog.f
	g77 -g -c myprog.f
```

Of course, I have no memory what it said for XSPEC 12.10.1

b) just try things randomly until it works